test/test_view.py

```diff
@@ -63,3 +63,29 @@ def test_populate_preview_image_missing_id(self):
         single_view._id = None
         single_view._workbook_id = '3cc6cd06-89ce-4fdc-b935-5294135d6d42'
         self.assertRaises(TSC.MissingRequiredFieldError, self.server.views.populate_preview_image, single_view)
+
+    def test_populate_image(self):
+        with open(POPULATE_PREVIEW_IMAGE, 'rb') as f:
+            response = f.read()
+        with requests_mock.mock() as m:
+            m.get(self.baseurl + '/views/d79634e1-6063-4ec9-95ff-50acbf609ff5/image', content=response)
```

Comment:
It's a mock, he's registering that URL so that when the API calls it it'll hit this fake endpoint and return the image